arrows/core/video_input_image_list.h

```diff
@@ -61,18 +65,19 @@ class KWIVER_ALGO_CORE_EXPORT video_input_image_list
                " in the \"image_reader\" config block."
                " Read an image list as a video stream." )
 
-  /// Constructor
   video_input_image_list();
   virtual ~video_input_image_list();
 
-  /// Get this algorithm's \link vital::config_block configuration block \endlink
-  virtual vital::config_block_sptr get_configuration() const;
+  /// Get this algorithm's
+  /// \link vital::config_block configuration block \endlink.
+  vital::config_block_sptr get_configuration() const override;
 
-  /// Set this algorithm's properties via a config block
-  virtual void set_configuration(vital::config_block_sptr config);
+  /// Set this algorithm's properties via a
+  /// \link vital::config_block configuration block \endlink.
+  void set_configuration( vital::config_block_sptr config ) override;
 
-  /// Check that the algorithm's currently configuration is valid
-  virtual bool check_configuration(vital::config_block_sptr config) const;
+  /// Check that the algorithm's currently configuration is valid.
+  bool check_configuration( vital::config_block_sptr config ) const override;
 
   /**
    * @brief Open a list of images.
```

Comment:
while you are fixing things, we should make this comment consistent with the style.  That is
```
/// Open a list of images.
/**
 * This method opens ...
```

Reply:
Is there a document anywhere explaining how documentation is supposed to be written? (The style guide just says "This section needs to be written".)

Reply:
All the code that came from MAP-Tk used the style of a single `\\\` line for the brief description followed by a `/**` block for the detailed description.  So this was the original KWIVER style.  Then we started to code added that uses `@brief` or `\brief` in a `/**`.  The MAP-Tk style was just my personal preference at the time.  I'm not strongly in favor of one style over another.  However, I am strongly in favor of being consistent.  We should pick something and put it in the style guide.  Then it would be nice to try to do a one time conversion that standardizes docs in our existing code.  That might be a good undergrad intern task.

Reply:
Fair enough. FWIW, my preference would be to use `///` and `\command` exclusively; I'm not a fan of multiple comment blocks with mixed style, and `///` is two fewer lines (and `@command` feels very Java).

Right now, I think the changes are no worse, and I was mostly resisting switching the existing `/**` to `///` to reduce churn, but what would you prefer?

Reply:
If we are make a KWIVER-wide style decision I would like input from at least @dstoup and @linus-sherrill.  Possibly this is something to discuss in a KWIVER cross-cutting meeting.  

What you are proposing is yet another doxygen style different than the mix we already have.  For now I would stick to one of the predominantly used styles in KWIVER and be consistent in this document.  I have no objections to your propose style, but consistency is the most important thing.  I would not introduce a new style unless you want to sign up for globally applying that style everywhere.